### PR TITLE
Changeling Armor Rebalance

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Lowers armor values (70% in most categories was absurd, especially with fleshmend). It now has 40% damage resist in most categories (and still no slowdown). Lings are running around tanking autorifles and beating everyone to death, it's just silly.

Still only costs 1 DNA point

### CODE REVIEW

File: code/game/gamemodes/changeling/powers/mutations.dm [hunk 1]
```diff
@@ -311,7 +310,7 @@
 	icon_state = "lingarmor"
 	flags = NODROP
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	armor = list(melee = 70, bullet = 50, laser = 70, energy = 50, bomb = 10, bio = 4, rad = 0)
```
Comment (Anonus) [inline code/game/gamemodes/changeling/powers/mutations.dm]: It's like old HoS armor, but on an antag with at-will regen!

**Comment by Lumipharon** [comment]
The chem recharge reduction thing or whatever it is, is a pretty significant balancing point.
It stops you from spamming fleshmend like every 10 seconds AND having fat ass damage resist.

File: code/game/gamemodes/changeling/powers/mutations.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/gamemodes/changeling/powers/mutations.dm]: Old HoS armor at least had a bullet weakness, this is closer to deathsquad

Comment (Iamgoofball) [inline code/game/gamemodes/changeling/powers/mutations.dm]: i dunno, I feel 50 should be the set

**Comment by KorPhaeron** [comment]
Won't touch that then, put it back

**Comment by GunHog** [comment]
Back to being useless and never bought again. Well, I never got to try it, and I suppose I never will. Bah.

**Comment by KorPhaeron** [comment]
I dunno how halving the damage you take is "useless" when you have the ability to unstun yourself and regenerate health.

Maybe it can be left at one DNA point.

**Comment by Lumipharon** [comment]
yeah 2 points is a little steep.

You really need the points to have the right powers, if you're going overt.

**Comment by Steelpoint** [comment]
The current Ling armour is even essentially 10 defence down from Death Squad armour.

I saw one Ling, from the same round, tank dozens of Auto Rifle rounds, a dozen Laser rounds and countless melee attacks. After wiping out a whole bunch of people in the HoP's office it took the Detective and a Assistant to finally stun, cuff and kill the ling.

Amusingly the ling only died due to the stunning properties of the Detective's Revolver allowing the Detective to cuff the ling.

**Comment by KorPhaeron** [comment]
There were 5 other lings as well, had any of them cared enough to show up to the fight (or had the guy on a killing spree asked for help first) there is little chance anyone would have killed htem.

**Comment by Steelpoint** [comment]
If all the Changelings in a round just went nuts and started killing everyone on station I don't doubt they would easily succeed.

**Comment by Nanotransistor** [comment]
What i would like is changelings having certain tiers of abilities, where they could choose either Fleshmend, armor, or adrenals, for example. Would balance the whole thing better.

**Comment by Anonus** [comment]
[Do you want lings to not be able to simultaneously stealth, murder, and annoy the crew?
Why would you ever want that?](https://tgstation13.org/phpBB/viewtopic.php?f=9&t=4158)

**Comment by Iamgoofball** [comment]
I'm okay with this PR. We test ran my armor changes and it went from UP to OP. This should be the nice middle point. :+1:

**Comment by Razharas** [comment]
Experienced firsthand - its bullshit

**Pull request merged**